docs/source/contributing/index.md

```diff
@@ -1,8 +1,12 @@
-# Contributing
+---
+title: "Contributing"
+sidebar_title: "Read This First"
+description: "placeholder"
```

Comment:
we don't need this "placeholder" in every doc. We just need a `description` in at least 1 `.md` file and at least 1 `.mdx` file.